makefiles/utils/strings.mk

```diff
@@ -0,0 +1,22 @@
+# String handling functions
+
+# Return an empty string if the arguments are different (works only for single
+# words
+streq = $(filter $(1),$(2))
```

Comment:
Even with only single words, it does not work. Using `findstring` would be better to not match patterns.
Here it could be checked that indeed `words $(1) == 1` and `words $(2) == 1`.